src/navigator/types.tsx

```diff
@@ -81,11 +81,11 @@ export type StackParamList = {
   [Screens.EarnInfoScreen]: undefined
   [Screens.EarnEnterAmount]: {
     pool: EarnPosition
-    mode?: EarnEnterMode
+    mode?: EarnActiveAction
   }
   [Screens.EarnConfirmationScreen]: {
     pool: EarnPosition
-    mode: WithdrawActionName
+    mode: EarnActiveAction
```

Comment:
same here, seems like this screen is specific to the withdraw actions

Reply:
Narrowed in ff2bd43cf3f9794d2b053f71d3ea04e459f2698f!